google/cloud/bigquery/job/base.py

```diff
@@ -171,6 +171,33 @@ def __setattr__(self, name, value):
             )
         super(_JobConfig, self).__setattr__(name, value)
 
+    @property
+    def jobtimeout(self):
```

Comment:
nit: normally we use snake_case to transform from rest API to python.

I'd prefer `job_timeout_ms` unless we change this to accept a `datetime.timedelta`, in which case `job_timeout` may make more sense.

Reply:
FWIW: We use `default_table_expiration_ms` here: https://github.com/googleapis/python-bigquery/blob/54a77694afcd80be4ba469c6ebb7ca8be112b04e/google/cloud/bigquery/dataset.py#L660 so probably makes the most sense to stay consistent with that.